### PR TITLE
The basic structure with Components and Store

### PR DESCRIPTION
**Pull request by javierbotero**
Using React and Redux this branch builds the basic structure with components connected with the react store. The user can add a book with a category and delete it too.

### CODE REVIEW

**Review by trekab — CHANGES_REQUESTED** [review]
### Changes Required! :recycle: 
Hi @javierbotero! You did a great job implementing most of the requirements for this milestone. However, there are a few changes for you to consider before moving forward. These include:
- [x] Create the initial state object in `src/index.js` ([screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/InitialBooks.png))
- [x] The `BookForm` component should be in the `src/containers` directory and it should have a constant with Books categories [screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/Form/CategoryConstants.png)
- [x] Please fix the several warnings that appear in the console when you run the app.

__N.B:__ Please note that it is important to focus on the specific milestone's requirements since these milestones build on each other to finally have the complete project.

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink: 
__Treasure Kabareebe__
[Portfolio](https://trekab.github.io/portfolio/) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

File: src/index.js [hunk 1]
```diff
@@ -1,8 +1,65 @@
...
+import App from './containers/App';
+import { FILTERS } from './constants/constants';
+import { CREATE_BOOK, REMOVE_BOOK } from './actions/index';
+import category from './reducers/categories';
+
+const initialState = [
+  {
+    title: 'A Hundred Years of Solitude',
+    category: FILTERS.biography,
+    id: Math.round(Math.random() * 10000),
```
Comment (Forison) [inline src/index.js]: * [ ] Math.random generates random number not unique numbers, for a robust unique number generation, we can use libraries like `shortid` or `uuid` these packages are built purposely for that. let's tweak the code a bit.

**Review by Forison — CHANGES_REQUESTED** [review]
# Status::Change required❌ 
#### Good job on addressing the issues raised in the previous review. All the three recommendations have been effected🤗 however there is a little tweak we have to make. 
* [x] Check my comment below.
#### Kindly make the change and submit a re-review😎
#### Happy coding
Addo Forison

**Review by Forison — CHANGES_REQUESTED** [review]
# Status::Approved✔️
### Good job on addressing the issue raised in the previous, go ahead merge the branch and proceed to the next milestone🤗🤗🤗.
#### Happy coding
Addo Forison

**Pull request merged**